src/handlebars/installation.handlebars

```diff
@@ -64,21 +79,236 @@
               <p>and confirm the command output succeeds.</p>
             </div>
 
-            <div style="margin-top: 3.0em;">
-              <h2 class="bold">Migration Guide</h2>
-              <p>
-                If you're migrating to AdoptOpenJDK, you can learn about the differences between Oracle JDK and OpenJDK in our
-                <a href="./migration.html" target="_blank">Migration Guide</a> and any steps that you might need to take.
-                For example, how to use IcedTea-Web as an alternative to Webstart.
-              </p>
             </div>
+          \{{/each}}
+        </script>
+      </div>
+    </div>
+    <div>
+      <br><br>
```

Comment:
Could apply some CSS to the hr tag or the div to create the vertical space as opposed to br tags?

Reply:
Applied class="margin-bottom" to the &lt;div&gt;